nasa-veda-platform/dashboard/index.qmd

```diff
@@ -1,21 +1,64 @@
 ---
-title: Dashboard
-subtitle: A graphical interface for discovering Earth data science
+title: VEDA Dashboard
+subtitle: 10 Things to Know About the VEDA Dashboard
 aliases:
   - /services/dashboard.html
 ---
 
-The [VEDA Dashboard](https://www.earthdata.nasa.gov/dashboard/) is one of several user interfaces in the VEDA project. The VEDA Dashboard is a publishing platform for results of scientific analysis of Earth data at NASA, 
-in the form of Dataset information pages ("Datasets") and stories ("Data Stories").
+The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. Learn more about the VEDA Dashboard set of tools below. If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit this page.
```

Comment:
```suggestion
The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. 

Learn more about the principles the VEDA Dashboard is designed around below.

If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit the [VEDA UI docs](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/development/SETUP.md).
```

Was this the page you meant to link to?

How would you describe the content of this page? I changed the sentence above to "the principles the VEDA Dashboard is designed around".